main/broadcasters.c

```diff
@@ -389,12 +388,78 @@ static broadcaster_ops_t mijia_temp_hum_ops = {
     .is_broadcaster = mijia_temp_hum_is_broadcaster,
     .metadata_get = mijia_temp_hum_metadata_get,
 };
+/* Beewi Smart Door */
+#define BEEWI_SMART_DOOR_COMPANY_ID     0x000D
+#define BEEWI_SMART_DOOR_SERVICE_ID     0x08 
+#define BEEWI_SMART_DOOR_DATA_TYPE_STAT 0x0C
+#define BEEWI_SMART_DOOR_DATA_TYPE_BATT 0x06
+
+
+typedef struct {
+    uint16_t company_id;
+    uint8_t  service_id;
+	uint8_t  type_stat;
+    uint8_t  status;
+	uint8_t  type_batt;
+	uint8_t  battery;
+} __attribute__((packed)) beewi_smart_door_t;
+
+static beewi_smart_door_t *beewi_smart_door_data_get(uint8_t *adv_data, uint8_t adv_data_len,
```

Comment:
Please make sure you don't exceed 80 characters per line (there are a few other places as well)